metaflow/plugins/kfp/s3_sensor_decorator.py

```diff
@@ -0,0 +1,77 @@
+from metaflow.decorators import FlowDecorator
+from metaflow.exception import MetaflowException
+
+from types import FunctionType
+from typing import Tuple
+from urllib.parse import urlparse
+
+"""
+Within identity_formatter, which is passed in as the path_formatter parameter,
+customers have access to all variables in flow_parameters_json (which 
+include all parameters passed in through the flow at compile and run time by
+KFP) as well as a number of environment variables.
+
+Please see metaflow/plugin/environment_decorator.py for details on how
+to add environment variables to be accessible within steps.
```

Comment:
can you please have an example of how to "access to a variable in flow_parameters_json"?